pyproject.toml

```
@@ -6,11 +6,11 @@ build-backend = "setuptools.build_meta"
 name = "lmfit"
 dynamic = ["version"]
 dependencies = [
-    "asteval>=0.9.28",
+    "asteval>=1.0",
     "numpy>=1.19",
     "scipy>=1.6",
     "uncertainties>=3.1.4",
```

```suggestion
    "uncertainties>=3.2.2",
```
?
(uncertainties was just released with some numpy 2.0 updates)

---

@eendebakpt Good suggestion, thanks!